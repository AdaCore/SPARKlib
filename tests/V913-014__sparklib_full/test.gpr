project Test extends "sparklib" is

   for Main use ("main.adb");

   package Compiler is
      for Default_Switches ("Ada") use
         ("-gnat2022", "-gnatygo-u",
	  "-gnatwI",  --  disable warnings about SPARKlib using GNAT internal units
          "-gnateDSPARK_BODY_MODE=" & SPARK_Body_Mode);
   end Compiler;

end Test;
