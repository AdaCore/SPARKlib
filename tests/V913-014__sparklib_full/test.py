from test_support import prove_all, gprbuild

prove_all(sparklib=True, opt=["-U"])
gprbuild(opt=["-P", "test.gpr"])
