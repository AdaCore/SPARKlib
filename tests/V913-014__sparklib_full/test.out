main.adb:25:13: info: implicit terminating annotation on "Hash" has been proved, subprogram will terminate
